govcd/tm_common_test.go

```diff
@@ -369,3 +371,41 @@ func createTmIpSpace(vcd *TestVCD, region *Region, check *C, nameSuffix, octet3
 		check.Assert(err, IsNil)
 	}
 }
+
+// vCenter task is sometimes unreliable and trying to refresh it immediately after it becomes
+// connected causes a "BUSY_ENTITY" error (which has a few different messages)
+var maximumVcenterRetryTime = 120 * time.Second                                         // The maximum time a single operation will be retried before giving up
+var vCenterEntityBusyRegexp = regexp.MustCompile(`(is currently busy|400|BUSY_ENTITY)`) // Regexp to match entity busy error
```

Comment:
What about pulling this parameter inside the func runWithRetry, so callers don't care?
Right now it is used in two files, which makes it kind-a hidden.

Reply:
Just realized the idea behind it. So, maybe instead of pulling it inside, it could be somewhere "higher" - more visible?

Reply:
We are going to use it in tests only so far. It will not be consumed in "production API"